modules/good-practices-lesson/documentation-slides.md

```diff
@@ -194,6 +207,9 @@ They follow a standardized syntax.
 
 ## Tools
 
++ **mkdocs** (documentation generator)
+  - creates nicely-formatted HTML pages out of .md files
+  - easy to use and deploy (e.g. GitHub pages)
```

Comment:
I guess mkdocs is also programming language indepentent

Reply:
Maybe we can say that there are various tools with mkdocs and sphinx being the most common ones, show their similarities and highlight differences, because now we repeate twice nearly the same